### PR TITLE
[PLAY-1046]  🛠️ Multilevel single select (phase 2)

### PR DESCRIPTION
**Pull request by Israel-Molestina**
**What does this PR do?** A clear and concise description with your runway ticket url.
Adds a hidden radio button functionality to the single select variant
[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-1046)

**Screenshots:** Screenshots to visualize your addition/change
Screenshot 2023-10-23 at 4 19 17 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to /kits/multi_level_select/rails
2. Scroll down to `Single Select`
4. See instances of single select labels without radio buttons

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx [hunk 1]
```diff
@@ -31,6 +32,7 @@ type MultiLevelSelectProps = {
   treeData?: { [key: string]: string }[]
   onSelect?: (prop: { [key: string]: any }) => void
   selectedIds?: string[]
+  // ultimateChildrenOnly?: boolean
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx:35]: Looks great @Israel-Molestina! Don't forget to remove the commented out code if you don't need it anymore!

File: playbook/app/pb_kits/playbook/pb_multi_level_select/multi_level_select.rb [hunk 2]
```diff
@@ -16,6 +16,8 @@ class MultiLevelSelect < Playbook::KitBase
                            default: "pills"
       prop :input_name, type: Playbook::Props::String,
                         default: ""
+      prop :ultimate_children_only, type: Playbook::Props::Boolean,
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_multi_level_select/multi_level_select.rb:19]: I see you removed this prop from the react side, I'm assuming we want to remove it from the rails side as well?

File: playbook/app/pb_kits/playbook/pb_multi_level_select/multi_level_select.rb [hunk 3]
```diff
@@ -33,6 +35,7 @@ def multi_level_select_options
           returnAllSelected: return_all_selected,
           selectedIds: selected_ids,
           input_name: input_name,
+          ultimateChildrenOnly: ultimate_children_only,
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_multi_level_select/multi_level_select.rb:38]: Same as above, let's remove this if we're no longer using it. Code looks great!

**Comment by nidaqg** [comment]
Code looks great! Some spacing/padding logic probably needs to be scoped more narrowly because currently the doc examples for several of the variants are a little wonky:

Screenshot 2023-11-08 at 2 25 03 PM


Screenshot 2023-11-08 at 2 24 48 PM

cc: @jasperfurniss

**Comment by Israel-Molestina** [comment]
> Code looks good @Israel-Molestina! Great work! Some spacing/padding logic probably needs to be scoped more narrowly because currently the doc examples for several of the variants are a little wonky:
> 
> Screenshot 2023-11-08 at 2 25 03 PM
> 
> Screenshot 2023-11-08 at 2 24 48 PM
> 
> cc: @jasperfurniss

Done! ready for review again

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**